content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -427,8 +427,17 @@ or updating objects that contain Pod templates, such as Deployments, Jobs, State
 See [Enforcing Pod Security at the Namespace Level](/docs/concepts/security/pod-security-admission)
 for more information.
 
-## seccomp.security.alpha.kubernetes.io/pod and container.seccomp.security.alpha.kubernetes.io/[NAME] (deprecated)
+## seccomp.security.alpha.kubernetes.io/pod (deprecated)
```

Comment:
nit:
```suggestion
## seccomp.security.alpha.kubernetes.io/pod (deprecated) {#seccomp-security-alpha-kubernetes-io-pod}
```

(tidier, but not a big deal as we have no plans to un-deprecate this annotation)